checkstyle-tester/diff.groovy

```diff
@@ -343,6 +351,7 @@ def printReportInfoSection(summaryIndexHtml, checkstyleBaseReportInfo, checkstyl
     summaryIndexHtml << ('<br />')
     summaryIndexHtml << "Patch branch last commit message: \"$checkstylePatchReportInfo.commitMsg\""
     summaryIndexHtml << ('<br />')
+    summaryIndexHtml << "Patch branch last commit timestamp: \"$checkstylePatchReportInfo.commitTime\""
     summaryIndexHtml << ('<br />')
```

Comment:
Missing new `<br />`. Original code had 2 blank lines instead of 1.

Reply:
Fixed.